.github/workflows/deploy.yml

```diff
@@ -254,7 +254,12 @@ jobs:
           git config user.email "${GIT_AUTHOR_EMAIL}"
           git config user.name "${GIT_AUTHOR_NAME}"
 
-      - name: Publish Docs Folder
+      - name: Update changelog
+        run: |
+          git checkout documentation docs/changelog/
+          yarn package-tools changelog --packages ${{ needs.generate-package-matrix.outputs.node-recursive }} --tag ${GITHUB_REF##*/} --commit ${{ github.event.before }}
+
+      - name: Update Changelog and Publish Docs Folder
```

Comment:
Are we not following any convention for the name description here? Somewhere we are using PascalCase and somewhere it's normal sentence.

Reply:
I think I dont think we are following any convention here. We can create a task to fix all these.